rerun_cpp/src/rerun/datatypes/rotation3d_ext.cpp

```diff
@@ -22,10 +22,6 @@ namespace rerun {
 #define Rotation3DExt Rotation3D
 #endif
 
-        // TODO(andreas): This constant initialization does not work for unknown reasons!
-        //                On clang(-mac) this set the Rotation3D::IDENTITY to all zero instead of
-        //                the expected quaternion.
-        // Using the same code as a non-constant works fine.
-        // const Rotation3DExt Rotation3DExt::IDENTITY = Quaternion::IDENTITY;
```

Comment:
This was generating one of the uninitialized union tag issues which was resolved by adding the constructor prior to the *this assignment. I suspect the underlying problem may have been the same but clang wasn't warning.